node/src/WebRtcTransport.ts

```diff
@@ -31,11 +32,19 @@ import { IceState as FbsIceState } from './fbs/web-rtc-transport/ice-state';
 export type WebRtcTransportOptions<WebRtcTransportAppData extends AppData = AppData> =
 	WebRtcTransportOptionsBase<WebRtcTransportAppData> & WebRtcTransportListen;
 
-export type WebRtcTransportListenIndividual =
+type WebRtcTransportListenIndividual1 =
+{
+	/**
+	 * Listening info.
+	 */
+	listenInfos: TransportListenInfo[];
+};
+
+type WebRtcTransportListenIndividual2 =
```

Comment:
Couldn't these be called `WebRtcTransportListenInfos` and `WebRtcTransportListenIps` ?

Reply:
Nope because the latter also includes an optional port so such a name would be confusing.

Reply:
Renamed here to more suitable names: https://github.com/versatica/mediasoup/pull/1065/commits/70662256731da0e2c4cd202811e9dab2a4c6ced2